### DISCUSSION TITLE
Remove Silence

### DISCUSSION THREAD

**Discussion by Melechtna** [audacity/audacity#6369] [Ideas & Feature Requests]
Basically what truncate silence already does, but instead of a trim operation, just cut these parts out entirely, and leave the audio where it already was. The upshot of which is the "silent" parts, become actually silent, removing things like soft hissing and such and often light breathing, while leaving the actual "meat" of the audio untouched.

**Comment by LWinterberg** [comment]
If you first use a noise gate and then edit > audio clips > detatch at silence, it does what you want it to do

**Reply by Melechtna** [reply]
Either I'm doing it wrong, or no it's doesn't. Noise gate first fills empty spaces, then the detatch at scilences removes the areas noise gate fills. I want to be absolutely certain there is NOTHING between the spoken areas, as this makes it more visually obvious if I've accidentally silenced an area I shouldn't have using said noisegate.
